spanner/google/cloud/spanner_v1/keyset.py

```diff
@@ -38,14 +38,15 @@ class KeyRange(object):
     """
     def __init__(self, start_open=None, start_closed=None,
                  end_open=None, end_closed=None):
-        if not any([start_open, start_closed, end_open, end_closed]):
-            raise ValueError("Must specify at least a start or end row.")
-
-        if start_open and start_closed:
+        if start_open is not None and start_closed is not None:
```

Comment:
It seems "wasteful" to do this `start_foo is not None` check in both `if` statements. What about:

```python
num_start = int(start_open is not None) + int(start_closed is not None)
```

(I had to write it down to see it's not any better than the current approach.)

Reply:
I don't really see how it's better.

Reply:
On this we agree